tests/functional/openstackdataplanedeployment_controller_test.go

```diff
@@ -30,23 +38,116 @@ var _ = Describe("Dataplane Deployment Test", func() {
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      nodeSet.Name,
+						Namespace: nodeSet.Namespace,
+					},
+				}
+				// Create config map for OVN service
+				Eventually(func(g Gomega) {
+					ovnConfigMapName := types.NamespacedName{
+						Namespace: namespace,
+						Name:      "ovncontroller-config",
+					}
+					mapData := map[string]interface{}{
+						"ovsdb-config": "test-ovn-config",
+					}
+					th.CreateConfigMap(ovnConfigMapName, mapData)
+				}).Should(Succeed())
+
+				// Set node set deployment ready condition to False
+				Eventually(func(g Gomega) {
+					g.Expect(th.K8sClient.Get(th.Ctx, dataplaneNodeSetName, &nodeSet)).To(Succeed())
+					nodeSet.Status.Conditions.MarkFalse(
+						condition.Type(fmt.Sprintf(dataplanev1.NodeSetServiceDeploymentReadyCondition, nodeSet.Name, "download-cache")),
+						condition.RequestedReason,
+						condition.SeverityInfo,
+						fmt.Sprintf(dataplanev1.NodeSetServiceDeploymentReadyMessage, nodeSet.Name, "download-cache"))
+					g.Expect(th.K8sClient.Status().Update(th.Ctx, &nodeSet)).To(Succeed())
+				}).Should(Succeed())
+
+				// Set baremetal provisioning conditions to True
+				Eventually(func(g Gomega) {
+					g.Expect(th.K8sClient.Get(th.Ctx, dataplaneNodeSetName, &baremetal)).To(Succeed())
+					baremetal.Status.Conditions.MarkTrue(
+						baremetalv1.OpenStackBaremetalSetBmhProvisioningReadyCondition,
+						baremetalv1.OpenStackBaremetalSetBmhProvisioningReadyMessage)
+					g.Expect(th.K8sClient.Status().Update(th.Ctx, &baremetal)).To(Succeed())
+
+				}).Should(Succeed())
+
+				// Create all services necessary for deployment
+				for _, serviceName := range nodeSet.Spec.Services {
+					dataplaneServiceName := types.NamespacedName{
+						Name:      serviceName,
+						Namespace: namespace,
+					}
+					CreateDataplaneService(dataplaneServiceName)
+					service := GetService(dataplaneServiceName)
+					//OVN needs config map resource
+					if serviceName == "ovn" {
+						service.Spec.ConfigMaps = append(service.Spec.ConfigMaps, "ovncontroller-config")
+
+						Eventually(func(g Gomega) {
+							g.Expect(k8sClient.Update(ctx, service)).To(Succeed())
+						}, th.Timeout, th.Interval).Should(Succeed())
+					}
+					//Retrieve service AnsibleEE and set JobStatus to Successful
+					Eventually(func(g Gomega) {
+						// Make an AnsibleEE name for each service
+						ansibleeeName := types.NamespacedName{
+							Name:      fmt.Sprintf("%s-%s", service.Spec.Label, dataplaneDeploymentName.Name),
+							Namespace: dataplaneDeploymentName.Namespace,
+						}
+						ansibleEE := &ansibleeev1.OpenStackAnsibleEE{
+							ObjectMeta: metav1.ObjectMeta{
+								Name:      ansibleeeName.Name,
+								Namespace: ansibleeeName.Namespace,
+							}}
+						g.Expect(th.K8sClient.Get(th.Ctx, ansibleeeName, ansibleEE)).To(Succeed())
+						ansibleEE.Status.JobStatus = ansibleeev1.JobStatusSucceeded
+
+						g.Expect(th.K8sClient.Status().Update(th.Ctx, ansibleEE)).To(Succeed())
```

Comment:
again this is OK to do as AnsibleEE is reconciled by the ansibleee-operator wich does not run in this env.